[OSDOCS#17175]: Make changes as per the errors and warnings for CQA-2 (#114273)
openshift/openshift-docs: modules/running-compliance-scans-worker-node.adoc

[id="running-compliance-scans-worker-node_{context}"]
=  Scheduling the result server pod on a worker node

[role="_abstract"]

> The result server pod mounts the persistent volume (PV) that stores the raw Asset Reporting Format (ARF) scan results. The `nodeSelector` and `tolerations` attributes enable you to configure the location of the result server pod.

Reviewer: Make this user-centric, not feature-centric. And add a _why_.

```suggestion
The result server pod mounts the persistent volume (PV) that stores the raw Asset Reporting Format (ARF) scan results. you can use the `nodeSelector` and `tolerations` attributes to configure the location of the result server pod to meet your organization's requirements.
```

  namespace: openshift-compliance
rawResultStorage:
  nodeSelector:
    node-role.kubernetes.io/worker: ""
  pvAccessModes:
  - ReadWriteOnce
  rotation: 3
  size: 1Gi
  tolerations:
  - operator: Exists
roles:
- worker
- master
scanTolerations:
  - operator: Exists
schedule: 0 1 * * *
----

> +

Reviewer 2: ```suggestion
+
where:
+
```

where:

> 

Reviewer 2: ```suggestion
```

`node-role.kubernetes.io/worker`:: Specifies the Compliance Operator uses this node to store scan results in ARF format.

> `operator`:: Specifies the result server pod tolerates all taints.

Reviewer: ```suggestion
`rawResultStorage.nodeSelector.node-role.kubernetes.io/worker`:: Specifies the Compliance Operator uses this node to store scan results in ARF format.
`rawResultStorage.tolerations.operator`:: Specifies the result server pod tolerates all taints.
```
